sdk/cosmosdb/cosmos/samples-dev/ItemManagement.ts

```diff
@@ -5,13 +5,12 @@
  * @summary Demonstrates item creation, read, delete and reading all items belonging to a container.
  */
 
-import * as dotenv from "dotenv";
-dotenv.config();
+import "dotenv/config";
 
-import { logSampleHeader, handleError, finish, logStep } from "./Shared/handleError";
-import { CosmosClient, PatchOperation, PriorityLevel } from "@azure/cosmos";
-
-import { Families } from "./Data/Families.json";
+import { logSampleHeader, handleError, finish, logStep } from "./Shared/handleError.js";
+import type { PatchOperation } from "@azure/cosmos";
+import { CosmosClient, PriorityLevel } from "@azure/cosmos";
+import Families from "./Data/Families.json" with { type: "json" };
```

Comment:
I've noticed that explicit imports have been introduced in many places. Is this the pattern recommended by the central team?

Reply:
Yes, it gives a clearer view to the developer that certain imports are type based and others are concrete.  This helps us run TypeScript directly in a number of environments such as using tsx, etc.